coordinator/src/main/scala/filodb.coordinator/queryplanner/MultiPartitionPlanner.scala

```diff
@@ -270,6 +270,15 @@ class MultiPartitionPlanner(partitionLocationProvider: PartitionLocationProvider
     (partitions, routingKeys)
   }
 
+  private def rvRangeFromLogicalPlan(logicalPlan: LogicalPlan, queryParams: PromQlQueryParams): Some[RvRange] = {
+    import LogicalPlan._
+    rvRangeFromPlan(logicalPlan) match {
+      case x @ Some(_)    =>  x
+      case None           => Some(RvRange(startMs = queryParams.startSecs * 1000L,
+        endMs = queryParams.endSecs * 1000L,
+        stepMs = queryParams.stepSecs * 1000L))
```

Comment:
The `RangeVector::outputRange` spec says:
```
...Present only for time series data that is periodic. If raw data is requested, then None.
```
Does this break for an instant query like `foo[8d]`?